Diff/Diff.sublime-syntax

```diff
@@ -24,7 +24,12 @@ first_line_match: |-
 contexts:
   main:
     - match: ^(?={{git_first_line}})
-      push: Packages/Git Formats/Git Diff.sublime-syntax#email-first-line
+      embed: Packages/Git Formats/Git Diff.sublime-syntax#email-first-line
```

Comment:
I thought of that, but I don't think those other syntaxes would be matching on the `git_first_line` pattern anyway.

I should probably update Git Log to use the new Git Diff syntax, but that is independent from this problem.

Reply:
We just want to forward to _Git Diff_ in case of certain patterns. Hence it's not the correct use case for an `meta.embedded` scope.

`meta.embedded` is or should primarily be used to scope evaluated scripts in template languages. Something like `<?php ...?>` or `{% ... %}` is meant to receive that scope.